gst/nnstreamer/include/nnstreamer_plugin_api_converter.h

```diff
@@ -74,6 +74,14 @@ typedef struct _NNStreamerExternalConverter
    */
 } NNStreamerExternalConverter;
 
+/**
+ * @brief Find converter sub-plugin with the name.
+ * @param[in] name The name of converter sub-plugin.
+ * @return NULL if not found or the sub-plugin object has an error.
```

Comment:
No @return description for success cases?

Reply:
Thanks! Updated desciption.